source/pdo_sqlsrv/pdo_util.cpp

```diff
@@ -409,6 +409,14 @@ pdo_error PDO_ERRORS[] = {
         SQLSRV_ERROR_SPECIFIED_DRIVER_NOT_FOUND,
         { IMSSP, (SQLCHAR*) "The specified ODBC Driver is not found.", -80, false }
     },
+    {
+        PDO_SQLSRV_ERROR_CE_DIRECT_QUERY_UNSUPPORTED,
+        { IMSSP, (SQLCHAR*) "Connection with Column Encryption enabled do no support PDO::SQLSRV_ATTR_DIRECT_QUERY with binding parameters.", -81, false }
+    },
+    {
+        PDO_SQLSRV_ERROR_CE_EMULATE_PREPARE_UNSUPPORTED,
+        { IMSSP, (SQLCHAR*) "Connection with Column Encryption enabled do no support PDO::ATTR_EMULATE_PREPARES with binding parameters.", -82, false }
+    },
```

Comment:
Please fix the two error messages: do _NOT_ support... 
Suggestion: Use English instead of the attribute names?
E.g. Emulate prepare is not supported when connecting with Column Encryption enabled. Something like that. I believe David might have a better idea.

Reply:
I will keep the attribute name since I think it's more clear for the user which attributes are not supported. But I'll change the error message according to your suggestions. :)

Reply:
I agree, keep the name of the attribute in the error message.